textproc/stanford-corenlp/Portfile

```diff
@@ -0,0 +1,255 @@
+        xinstall_recursive_dir ${worksrcpath} ${dest_doc} scripts
+        xinstall -m 0644 -W ${worksrcpath} \
+            LICENSE.txt README.md RESOURCE-LICENSES \
+            ${dest_doc}    
+    }
+}
+
+# individual language models
+proc stanford-corenl-models-setup {} {
+    global name subport version
+
+    if {${subport} ne "${name}-models"} {
+        depends_run-append \
+                    port:stanford-corenlp
+    }
+
+    master_sites    https://repo.maven.apache.org/maven2/edu/stanford/nlp/${name}/${version}/
+    set subport_tag \
+                    [strsed ${subport} "s|${name}-||"]
+    if { [string match "models" ${subport_tag}] } {
+        distname        ${name}-${version}-${subport_tag}
+    } else {
+        distname        ${name}-${version}-models-${subport_tag}
+    }
+    extract.suffix      .jar
+    extract.only
+    dist_subdir         ${subport}
+    build {}
+    destroot {
+        # install jar files, scripts, classes, etc.
+        xinstall -m 0755 -d ${dest_java}
+        xinstall -m 0644 -W ${distpath} ${distname}.jar ${dest_java}
+    }
+}
+
+subport ${name}-models {
+    stanford-corenl-models-setup
+    checksums       rmd160  94e4d75bdd0f496a765e681bd60e0f5a29b33872 \
+                    sha256  5997eeebfd631f9eb779c1d4246ac5fcf09298ffbf6db5e00c0f1b80dc34ba10 \
+                    size    362565193
+}
+
+subport ${name}-arabic {
+    stanford-corenl-models-setup
+    checksums       rmd160  ecf19e0f6a4d8ffdad228cb843178b491bfd55f5 \
+                    sha256  7d102f4586731a4b15e8f109f879286790e74e6bdc3eb2e4ab09e99fd7375ba1 \
+                    size    65787818
+}
+
+subport ${name}-chinese {
+    stanford-corenl-models-setup
+    checksums       rmd160  14d3964b8840ecc70db938b0e8c2194ac6d3bb49 \
+                    sha256  22221e19ca2d8d4ac061d900b6432d6c2fc9b030a647d91d8b8cc9954b59053b \
+                    size    1016572179
+}
+
+subport ${name}-english {
+    stanford-corenl-models-setup
+    checksums       rmd160  827c59a32f995f2ec4c7cff3929e16b565854d34 \
+                    sha256  dbd1443383d2bd77f0fa083d04db33465445b714a5532cd57abb88ff7db7580c \
+                    size    1038970602
+}
+
+subport ${name}-english-kbp {
+    stanford-corenl-models-setup
+    checksums       rmd160  b9f6ebe61dbce90eacbb4231c5192acb6e151b5b \
+                    sha256  a10d22a266221f3fb34796847839a98462c1dd3e0b8ab34167768f21530278ce \
+                    size    473999624
+}
+
+subport ${name}-french {
+    stanford-corenl-models-setup
+    checksums       rmd160  fa7fb4fc3657f23fe6c4f170c7049021b7a35f58 \
+                    sha256  d2f5364cd00add931fe3346435051c091f9a005a415bbb54f8a8a67b42172c83 \
+                    size    272154566
+}
+
+subport ${name}-german {
+    stanford-corenl-models-setup
+    checksums       rmd160  cc9ffa8d823105cbcc543249b15eb9feba9ec094 \
+                    sha256  d992669d8d3a5b64f1fd92048d37e3f5936752c7408b4edf8d226a1a7ebb6e9f \
```

Comment:
It looks like the `size` got left out here.

Reply:
Done—thanks.